### PR TITLE
HTML CSS FILES

### PR DESCRIPTION
**Pull request by CakeInTech**
Added an HTML file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
Added a css file that includes one class with styles for your header (make it in your favorite color).
Added test.md file with the text "This file should be ignored by git"
Used linter on the GitHub project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
-# Hell-Microverse-Project
\ No newline at end of file
+# HTML & CSS3 Course
```
Comment (tufoinnkuo10) [inline README.md]: - [x] kindly use this readme template. [readme](https://github.com/microverseinc/readme-template)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @MohamedCK 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated

## Required Changes ♻️
Check the comments under the review
Optional suggestions
- [ optional] please next time do not open a new pull request unless the current one has been approved by a code reviewer.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (CakeInTech) [reply README.md]: > * [x]  kindly use this readme template. [readme](https://github.com/microverseinc/readme-template)
Ok i used a readme template

Comment (CakeInTech) [reply README.md]: @tufoinnkuo10 what do you mean by every comment prefix pls

Comment (tufoinnkuo10) [reply README.md]: it is just a reminder that any comment/requested change with optional in it won't stop the approval of your pull request

Comment (CakeInTech) [reply README.md]: Oh ok thanks @tufoinnkuo10

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @MohamedCK  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Made changes as requested by the previous reviewer 👍 
## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**